src/app/resources/iis_scripts/admin_api_util.ps1

```diff
@@ -212,73 +218,74 @@ if (!$config.security.users.owners.Contains($user)) {
         $config.security.users.administrators += $iisAdminOwners
         $saveConfig = $true
     }
+}
+
 
-    if ($saveConfig) {
-        LogVerbose "Saving config..."
-        if ($devMode) {
-            throw "Cannot edit config file in dev mode, please add $user to ""$iisAdminOwners"" group manually"
+if ($saveConfig) {
+    LogVerbose "Saving config..."
+    if ($devMode) {
+        throw "Cannot edit config file in dev mode, please add $user to ""$iisAdminOwners"" group manually"
+    }
+    if (!(Get-LocalGroupMember -Group "Administrators" -Member $user -ErrorAction SilentlyContinue)) {

```

Comment:
One globalization issue here. We can't use the hard-coded name for "Administrators" local group. In Germany, it is a different one. Please find out how to solve the issue. FYI, I found one article about that. https://stackoverflow.com/questions/3212862/how-can-i-get-the-local-group-name-for-guests-administrators

Reply:
One more article. 
https://stackoverflow.com/questions/24426405/find-admin-group-name

Reply:
That's a good point. But this is a different work item. Can you open a new issue and I will fix it. This PR is just for CORS